test/cluster_client_replicas_test.rb

```diff
@@ -13,10 +13,7 @@ def test_client_can_command_with_replica
       assert_equal 'OK', r.set("key#{i}", i)
     end
 
-    begin
-      r.wait(6, 5_000)
-    rescue Redis::TimeoutError
-    end
+    r.wait(1, TIMEOUT.to_i * 1000)
```

Comment:
The `WAIT` command is sent to all master nodes.
https://github.com/redis/redis-rb/blob/v4.1.1/lib/redis/cluster.rb#L135

The master node has a replica. So I think the `1` as first argument is correct.
```
TestClusterClientReplicas#test_some_reference_commands_are_sent_to_slaves_if_needed = 15.13 s = F
```
The `6` as the number of replicas per master node spent `5 * 3` seconds.